toolkit/tools/imagecustomizerschemacli/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+BINARY_NAME := bin/imagecustomizerschemacli
+GO_FILES := $(wildcard *.go)
+SCHEMA_FILE := schema.json
```

Comment:
Should this `schema.json` be checked-in?

Reply:
Yes, I'm planning to but will do that in a separate change. The other change will include adding a github workflow to ensure this file is kept up to date with any change/PR.